app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/HealthRecordsFragment.kt

```diff
@@ -72,25 +68,35 @@ class HealthRecordsFragment : Fragment(R.layout.fragment_health_records) {
         }
     }
 
+    private fun initUi() {
+        binding.ivAddHealthRecord.visibility = View.VISIBLE
+        binding.tvTitle1.visibility = View.VISIBLE
+        binding.tvMessage.visibility = View.VISIBLE
+        setupToolBar()
+    }
+
     private suspend fun collectHealthRecordsFlow() {
         viewModel.patientHealthRecords.collect { records ->
+            binding.progressBar.isVisible = false
             if (records.isNotEmpty()) {
-                binding.ivAddHealthRecord.visibility = View.VISIBLE
-                binding.rvMembers.adapter = adapter
-                if (records.any { it.authStatus == AuthenticationStatus.AUTHENTICATED }) {
-                    binding.rvMembers.layoutManager = GridLayoutManager(requireContext(), 2).apply {
-                        spanSizeLookup =
-                            object : GridLayoutManager.SpanSizeLookup() {
-                                override fun getSpanSize(position: Int) = when (position) {
-                                    0 -> GRID_SPAN_COUNT
-                                    else -> LIST_SPAN_COUNT
-                                }
-                            }
-                    }
+                if (records.size == 1) {
+                    val navOptions = NavOptions.Builder()
```

Comment:
After adding any covid or vaccine public record successfully and when you come back to this screen does it update newly added record folder ? Please test this scenario.

Reply:
this will be taken care